ci/run_easymesh_cert.sh

```diff
@@ -24,6 +24,7 @@ usage() {
     echo "      -v|--verbose - set verbosity (ucc logs also redirected to stdout)"
     echo "      -o|--log-folder - path to put the logs to (make sure it does not contain previous logs)."
     echo "      -e|--easymesh-cert - path to easymesh_cert repository (default ../easymesh_cert)"
+    echo "      --reset-easymesh-cert - reset the easymesh_cert repository to the version specified in this file."
```

Comment:
I think it will be useful to allow the user to select the revision using another option - `--easymesh-cert-revision`, and if it is not selected use the default in this file.

Reply:
I have the impression that when you want to try things out locally, you'd prefer to use uncommitted changes, so not using the option at all. 
We can still add the option later if it turns out someone need it.